examples/insurance/flows/customEnrichment.flow.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name" : "customEnrichment",
+  "description" : "custom code example to enrich data",
+  "batchSize" : 100,
+  "threadCount" : 4,
+  "stopOnError" : false,
+  "options" : { },
+  "version" : 0,
+  "steps" : {
+    "1" : {
+      "name" : "ingestAdvantage",
+      "description" : "ingesting Advantage insurance customers",
+      "options" : {
+        "additionalCollections" : [ ],
+        "headers" : {
+          "sources" : [ {
+            "name" : "customEnrichment"
+          } ],
+          "createdOn" : "currentDateTime",
+          "createdBy" : "currentUser"
+        },
+        "sourceQuery" : "cts.collectionQuery([])",
+        "collections" : [ "ingestAdvantage" ],
+        "permissions" : "rest-reader,read,rest-writer,update",
+        "outputFormat" : "json",
+        "targetDatabase" : "insurance-STAGING"
```

Comment:
I cloned this locally, and in QuickStart, I had to initialize the project - that's because gradle.properties isn't checked in. That results in the default database names being used. When I run this flow, I get an error because "insurance-STAGING" doesn't exist. So I think this example project should assume that "data-hub-STAGING" and the other default names will be used as well.

Reply:
Agree. I will make that change so it uses default app name.

Reply:
I will be pushing changes in a bit without the gradle.properties and let QS initialize the project. So that whenever some tries it they will have relevant project properties initialized at the time. This also avoids us having to update the gradle.properties with mlDHFVersion every release.